apps/dashboard/src/components/workflow-editor/base-node.tsx

```diff
@@ -66,7 +66,7 @@ export const NodeBody = ({ children }: { children: ReactNode }) => {
       <span className="text-foreground-400 overflow-hidden text-ellipsis text-nowrap text-sm font-medium">
         {children}
       </span>
-      <span className="to-background/90 absolute left-0 top-0 h-full w-full bg-gradient-to-r from-[rgba(255,255,255,0.00)] from-70% to-95%" />
+      <span className="to-background/90 absolute left-0 top-0 h-full w-full rounded-b-[calc(var(--radius)-1px)] bg-gradient-to-r from-[rgba(255,255,255,0.00)] from-70% to-95%" />
```

Comment:
why the -1px ?

Reply:
because the outer container has 1px padding,so (outer radius - padding) = inner radius